content/develop/ai/index.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: Redis for AI documentation
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+- rc
+description: An overview of the Redis for AI documentation
+linkTitle: Redis for AI
+weight: 40
+---
+Redis stores and indexes vector embeddings that semantically represent unstructured data including text passages, images, videos, or audio. Store vectors and the associated metadata within hashes or JSON documents for indexing and search.
+
+| Vector | RAG |
+| :-- | :-- |
+| {{<image filename="images/ai-cube.png" alt="AI Redis icon.">}}[Redis as a vector database quick start guide]({{< relref "/develop/get-started/vector-database" >}}) |{{<image filename="images/ai-brain.png" alt="AI Redis icon.">}} [Retrieval-Augmented Generation quick start guide]({{< relref "/develop/get-started/rag" >}}) |
+
+
+## Overview
+
+1. [**Create a vector index**]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#create-a-vector-index): Redis maintains a secondary index over your data with a defined schema (including vector fields and metadata). Redis supports [`FLAT`]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#flat-index) and [`HNSW`]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#hnsw-index) vector index types.
+1. [**Store and update vectors**]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#store-and-update-vectors): Redis stores vectors and metadata in hashes or JSON objects.
+1. [**Search with vectors**]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#search-with-vectors): Redis supports several advanced querying strategies with vector fields including k-nearest neighbor ([KNN]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#knn-vector-search)), [vector range queries]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#vector-range-queries), and [metadata filters]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#filters).
+1. [**Configure vector queries at runtime**]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#runtime-query-parameters).
+
+## Resources 
+
+Learn to perform vector search and use Redis integrations in your AI/ML projects.
+
+| Search | Cloud Model Integration | Python Vector Library | 
+| :-- | :-- | :-- |
+| {{<image filename="images/ai-search.png" alt="AI Redis icon.">}}[Vector search guide]({{< relref "/develop/interact/search-and-query/query/vector-search" >}}) | {{<image filename="images/ai-model.png" alt="AI Redis icon.">}}[Amazon Bedrock setup guide]({{< relref "/integrate/amazon-bedrock/set-up-redis" >}}) |{{<image filename="images/ai-brain-2.png" alt="AI Redis icon.">}}[Redis vector python library documentation]({{< relref "/integrate/redisvl/overview/" >}})
+
+## Next steps
```

Comment:
Just thinking it might work better for the right-hand side nav menu. If you are going to add stuff other than notebooks here eventually then I guess Next steps is better.
```suggestion
## Notebooks
```